content/docs/0.8.1/volumes-and-nodes/scheduling.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Scheduling
+weight: 8
+---
+
+In this section, you'll learn how longhorn schedules replicas based on multiple factors.
+
+#### Scheduling Policy
+
+Longhorn scheduling policy has several layers, only the previous layer gets satisfied, the scheduler will goto the next layer, otherwise the scheduling will fail.
+
+1. Node and Zone selection layer, Longhorn will filter the node and zone based on `Replica Node Level Soft Anti-Affinity` and `Replica Zone Level Soft Anti-Affinity`.
+2. Tag selection layer, Longhorn will filter the node which satisfied the first layber with correct tag.
+3. Disk selection layer, Longhorn will filter the disk which satisfied the first two layers based on `Storage Minimal Available Percentage`, `Storage Over Provisioning Percentage` and other disk related factors like request disk space. 
+
+##### Node and Zone Selection Layer
+
+First, Longhorn will always try to schedule the new replica on a new node with a new zone if possible. At this time if both `Replica Node Level Soft Anti-Affinity` and `Replica Zone Level Soft Anti-Affinity` are un-checked, this means hard on node and hard on zone. Longhorn will not schedule the replica if there is no new node with new zone.
```

Comment:
Don't say `hard on node and hard on zone` (and same below). It's not proper documentation language.